### ISSUE TITLE
Set up Jenkins pipeline to create build artifacts

### ISSUE THREAD

**Issue by TrevCraw** [OpenLiberty/liberty-tools-intellij#106]
Jenkins should be triggered to create build artifacts when PRs are opened and when PRs are merged into the main branch.
The resulting artifact from this build will be the `.zip` file, which is created by the command `./gradlew buildPlugin`.
Gradle is a requirement for the build environment.

Once the `ls-integration` branch is merged into main, there will be a build dependency on the Liberty LS, LSP4MP, and LSP4Jakarta. Liberty LS and LSP4MP can be handled through pulling from Sonatype or Maven Central using `build.gradle`. 

The build process will need to be resolved for LSP4Jakarta before working on that piece.

**Comment by TrevCraw** [comment]
Opened new issue to handle language servers piece - https://github.com/OpenLiberty/liberty-tools-intellij/issues/118
